libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -217,42 +228,14 @@ class CodeTransform: public boost::static_visitor<>
 		int height = m_state.assembly.deposit();
```

Comment:
I think this PR is not about the safer language.  I'm basically trying to point out ways in which this less-safe language can go into inconsistent states.  I'm also assuming we would want to remove such possibilities.